CHANGELOG.md

```diff
@@ -59,6 +61,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add orders endpoints ([#212](https://github.com/whirli/whirli-client/pull/212))
 - Add checkout endpoint ([#213](https://github.com/whirli/whirli-client/pull/213))
 - Add shipping addon resource and get all endpoint ([#214](https://github.com/whirli/whirli-client/pull/214))
+### 🐛 Bug fixes
+- Change returnOrder complete from PUT to POST to test image uploading ([#195](https://github.com/whirli/whirli-client/pull/195))
```

Comment:
Why is this added here?

Reply:
I had a crazy merge conflict, let me have a look at whats in develop

Reply:
this is fixed now